aws_helper/config.go

```diff
@@ -28,3 +29,26 @@ func CreateAwsSession(awsRegion, awsProfile string) (*session.Session, error) {
 
 	return session, nil
 }
+
+// AssumeRoleEnvironmentVariables returns a set of key value pair to use as environment variables to assume a different role
+func AssumeRoleEnvironmentVariables(roleArn string, sessionName string) (map[string]string, error) {
+	session, err := CreateAwsSession("", "")
```

Comment:
Let me recap with our current situation:
- We currently have an IAM user in the AWS account A
- That user is allowed to assume the developer role in an account B
- The developer role is allowed to assume the deployment role in the account B
- The IAM user in the account A is currently not allowed to assume the deployment role in the account B

So, with our current access policies, it will not work since the CreateAwsSession method here will use the access key of the IAM user of the AWS account A that cannot assume the deployment role of the account B. 

I can fix all that if I allow the IAM user in the AWS account A to assume the deployment role in the account B.

So with your fix, we have to do two more things:
- Set a trust policy on your deployment role in the account B to allow the IAM user in the account A to assume it
- Add the statement to allow the IAM user in the account A to assume the role in the account B

Is this what you have in mind?